yaml/pois.yaml

```diff
@@ -87,19 +87,28 @@ global:
         - US National Park service
   - &no_name_okay
         - aerialway: pylon
-        - aeroway: gate
-        - amenity: [ bbq, bicycle_parking, bicycle_repair_station, boat_storage,
-            life_ring, parking, picnic_table, ranger_station, shower, toilets,
-            waste_disposal, water_point, watering_place ]
+        - aeroway: [gate,helipad]
+        - amenity: [ atm, bbq, bench, bicycle_parking, bicycle_rental,
+            bicycle_repair_station, boat_storage, car_sharing, fuel,
+            life_ring, parking, picnic_table, post_box, ranger_station, recycling,
+            shelter, shower, telephone, toilets, waste_basket, waste_disposal,
+            water_point, watering_place ]
         - barrier: [ cycle_barrier, gate, toll_booth ]
         - emergency: [ lifeguard_tower, phone ]
-        - highway: [ motorway_junction, rest_area, trailhead ]
+        - highway: [ bus_stop, ford, mini_roundabout, motorway_junction, platform,
+            rest_area, traffic_signals, trailhead ]
         - historic: landmark
-        - leisure: [ firepit, fishing, pitch, slipway, swimming_area ]
-        - man_made: [ communications_tower, mast, observatory, petroleum_well,
-            water_tower, water_well, windmill ]
-        - natural: [peak, volcano, geyser, hot_spring, saddle, spring, tree ]
+        - landuse: quarry
+        - leisure: [ dog_park, firepit, fishing, pitch, slipway, swimming_area ]
+        - lock: yes
+        - man_made: [ adit, communications_tower, mast, mineshaft, observatory,
```

Comment:
The addition of `mineshaft` here seems to have made a [previous test](https://github.com/tilezen/vector-datasource/blob/bc8c87e7a361ed7424af1cff39390462514b75c0/integration-test/1186-the-pois-with-no-name.py#L29) fail. There's probably a whole load more of these given all the extra PONIs that are being allowed. Might it be shorter and more readable to switch this meaning of the list around and blacklist those for which no name is not allowed?